src/components/Animator/AnimatorComponent.tsx

```diff
@@ -235,6 +237,7 @@ export class AnimatorComponent extends React.Component<WidgetProps> {
         if (numChannels > 1) {
             const numLabels = 5;
             const channelStep = numChannels > 10 ? Math.floor((numChannels - 1) / (numLabels - 1)) : 1;
+            const tickChannels = numChannels > 10 ? [0, channelStep, 2 * channelStep, 3 * channelStep, 4 * channelStep, numChannels - 1] : Array.from(Array(numChannels).keys());
```

Comment:
with a large number of channels, the last two labels may be too close.
<img width="369" alt="image" src="https://user-images.githubusercontent.com/43841102/177454421-e8ba3b23-af73-4dc4-9a8b-19f73174b872.png">

Reply:
Would like to discuss a bit about this issue with @YuHsuan-Hwang  and @kswang1029.
The easiest way of solving this issue is to show only 5 ticks (instead of the current 6).
With this modification, the showing ticks are: 
0, channelStep, 2*channelStep, 3*channelStep, and numChannels - 1 (the last index).
It's the best method I can think of so far to show the necessary ticks (most importantly, the last index) and get rid of the congestion of the last two indices in most cases.
Nevertheless, it may produce wierd pattern of slider when there are 11 or 12 elements to be shown on the slider, something like this:
<img width="343" alt="image" src="https://user-images.githubusercontent.com/106953609/177464406-669ec977-4064-4a5f-b83d-f2f3f7fdf3c0.png">
Is it acceptable for you? I would like to take comments if you have better solutions.

Reply:
perhaps check the difference between `4 * channelStep, numChannels - 1` and remove `4 * channelStep` when needed?

Reply:
Thanks for the tip.
I made some modifications following your suggestion, but adopt different methodology for frame (image) and channel, respectively. The difference is shown below:
Frame:
<img width="738" alt="image" src="https://user-images.githubusercontent.com/106953609/177516173-05560986-d0d5-4cd0-b37b-bdd4cd587c20.png">
Channel
<img width="739" alt="image" src="https://user-images.githubusercontent.com/106953609/177516283-36aca2e6-a806-4352-a314-56d19fbb7497.png">
The reason is that we usually deal with great numbers of channel (hundreds?) while there are usually much less of images (tens for most?). Although I do think that my resolution is a bit tricky and somewhat too custormized, it seems to be the best configuration so far.
Please take a look of my last commit.

Reply:
Thank you! Should have thought about it.
The new commit is established.